### ISSUE TITLE
Add property to retrieve hash only

### ISSUE THREAD

**Issue by bsoyka** [bsoyka/gravify#7]
All that is needed is a new property under the `Gravatar` class to retrieve the md5 hash of `self.email`.
___
This line can retrieve the needed hash: (found in the `url` property)
```py
hashlib.md5(self.email.encode("utf-8").lower()).hexdigest()
```
___
Also, under the `url` property, you can replace the hash part of the url with a call to the new property. (example: `self.hash`)

**Comment by charanjit-singh** [comment]
I'd like to work on it

**Comment by bsoyka** [comment]
Sounds great! Make a pull request once you’re done, and let me know if you have any questions! 😃

**Comment by bsoyka** [comment]
@charanjit-singh Do you still plan on working on this?

**Comment by charanjit-singh** [comment]
Yes. I'm on it. Apologies for delay

**Comment by bsoyka** [comment]
No worries, I just wanted to make sure. 😄

**Comment by bsoyka** [comment]
Closed by #12
